components/TreasuryAccount/NewTreasuryAccountForm.tsx

```diff
@@ -53,7 +53,7 @@ const defaultFormValues = {
   minInstructionHoldUpTime: 0,
   maxVotingTime: 3,
   voteThreshold: 60,
-  voteTipping: VoteTipping.Strict,
```

Comment:
- The UI should ask for 'Council Vote Tipping' and set it accordingly (please update label in the form as well). The default value in the form should be `Strict` 
- The Community Vote Tipping should always be set to `Disabled`